pkg/capabilities/capabilities.go

```diff
@@ -112,35 +115,42 @@ func MergeCapabilities(base, adds, drops []string) ([]string, error) {
 		return nil, err
 	}
 
-	// Make sure that capDrop and capAdd are distinct sets.
+	if stringInSlice(All, capDrop) {
+		// "Drop" all capabilities; return what's in capAdd instead
+		return capAdd, nil
```

Comment:
if capAdd is ALL, should we instead return capabilityList here?

Reply:
The idea is if a user says --cap-drop all  and then adds a few capabilities, then he gets those caps.  So this is correct

Reply:
Right, what I'm trying to guard against is someone doing the goofy case, "All" in both drop and add.  In that odd case, then they'd get back All caps, but maybe too weird to worry about.